devfiles/go/devfile.yaml

```diff
@@ -9,10 +9,21 @@ projects:
     type: git
     location: https://github.com/golang/example.git
   clonePath: src/github.com/golang/example/
+-
+  name: golang-echo-realworld-example-app
+  source:
+    type: git
+    location: https://github.com/xesina/golang-echo-realworld-example-app.git
+  clonePath: src/github.com/xesina/golang-echo-realworld-example-app
+
 components:
+-
+  type: cheEditor
+  id: eclipse/che-theia/next
+  memoryLimit: 1Gi
 -
   type: chePlugin
-  id: ms-vscode/go/latest
+  id: golang/go/latest
   alias: go-plugin
   memoryLimit: 512Mi
```

Comment:
Does the plugin need any more memory than before? Or can we leave it?

Reply:
The first I gave it 1Gb, but then reduced to the previous value.